### ISSUE TITLE
Unit tests failing on python 2.7

### ISSUE THREAD

**Issue by antonl** [saullocastro/cubature#12]
A few unit tests are failing when running the test suite under python 2.7. This turns out to be because of integer division. This is not a problem under python 3. 

```
$ py.test                                                                                                                                    (cython-2p7) 
================================================================== test session starts ===================================================================
platform darwin -- Python 2.7.9, pytest-2.8.1, py-1.4.30, pluggy-0.3.1
rootdir: /Users/aloukian/Documents/cubature, inifile: 
plugins: xdist-1.9
collected 18 items 

test_cubature.py FFFF..........
test_internal.py ....

======================================================================== FAILURES ========================================================================
_____________________________________________________________ test_genz_oscillatory_exact_d3 _____________________________________________________________

    def test_genz_oscillatory_exact_d3():
        exact = 4 * np.cos(1056 + np.pi/7)*np.sin(12)*np.sin(306)*np.sin(738)/6273
        calculated = ti.genz_oscillatory_exact(12, np.array([2, 123, 51]), 1/14)
>       assert np.allclose([exact], [calculated])
E       assert <function allclose at 0x1035e1500>([5.6744864646321004e-05], [8.0526955541289995e-05])
E        +  where <function allclose at 0x1035e1500> = np.allclose

test_cubature.py:13: AssertionError
_____________________________________________________________ test_genz_oscillatory_exact_d4 _____________________________________________________________

    def test_genz_oscillatory_exact_d4():
        exact = np.sin(29988 - 7*np.pi/34) * \
                np.sin(51)*np.sin(2601/2)*np.sin(6273/2)*np.sin(25500) / \
                784125
        calculated = ti.genz_oscillatory_exact(51, np.array([2, 123, 51, 1000]),
                11/17)
>       assert np.allclose([exact], [calculated])
E       assert <function allclose at 0x1035e1500>([7.1966992321033331e-08], [2.0775359271081527e-09])
E        +  where <function allclose at 0x1035e1500> = np.allclose

test_cubature.py:21: AssertionError
________________________________________________________________ test_genz_oscillatory_d2 ________________________________________________________________

    def test_genz_oscillatory_d2():
        x = np.array([17, 201], dtype=float)
        a = np.array([41, 1/11], dtype=float)
        u = 1/51
    
        exact = np.cos(7868/11 + 2*np.pi/51)
        calculated = ti.genz_oscillatory(x, a, u)
>       assert np.allclose([exact], [calculated])
E       assert <function allclose at 0x1035e1500>([0.39940800746348032], [0.9074721111282797])
E        +  where <function allclose at 0x1035e1500> = np.allclose

test_cubature.py:30: AssertionError
_______________________________________________________________ test_genz_oscillatory_c_d2 _______________________________________________________________

    def test_genz_oscillatory_c_d2():
        x = np.array([17, 201], dtype=float)
        a = np.array([41, 1/11], dtype=float)
        u = 1/51
    
        exact = np.cos(7868/11 + 2*np.pi/51)
        calculated = ti.genz_oscillatory(x, a, u)
        calculated_2 = ti.genz_oscillatory_c(x, a, u)
    
>       assert np.allclose([exact], [calculated_2])
E       assert <function allclose at 0x1035e1500>([0.39940800746348032], [0.9074721111282797])
E        +  where <function allclose at 0x1035e1500> = np.allclose

test_cubature.py:41: AssertionError
================================================ 4 failed, 14 passed, 2 pytest-warnings in 516.79 seconds ===============================================
```

Here is a diff that makes the unit tests pass
```
$ git diff --cached test_cubature.py                                                                                                         
diff --git a/test_cubature.py b/test_cubature.py
index 57137f7..99fd165 100644
--- a/test_cubature.py
+++ b/test_cubature.py
@@ -1,3 +1,4 @@
+from __future__ import division, print_function
 import pytest
 
 import cubature._test_integrands as ti
``` 

This just needs a patch.

**Comment by antonl** [comment]
Somehow the from future division line was removed.

**Comment by saullocastro** [comment]
I can see it right here https://github.com/saullocastro/cubature/blob/master/test_cubature.py#L1

**Comment by antonl** [comment]
Fixed with @f2a837ab. 
@saullocastro I just pushed that. Should have closed the commit from the commit message.
